packages/core/nuxt-theme-module/theme/components/AppHeader.vue

```diff
@@ -91,13 +184,37 @@ export default {
   --header-padding:  var(--spacer-sm);
   @include for-desktop {
     --header-padding: 0;
+    z-index: var(--header-wrapper-z-index, 1);
```

Comment:
please test it with sidebars and bottom nav

Reply:
I didn't see any problems